README.md

```diff
@@ -388,6 +388,7 @@ API documentation is generated when `npm install` is run.
 # Release Notes
 
 ## Version 1.3.1-1
+* SEMI-1054 Adds tipAmount to tipAdded 
```

Comment:
This is out of date.  The current version is 1.3.1-2.
The issue needs to go above the 1.3.1-2 ## Version 1.3.1-2 tag.  The change will be after this, as it is already released.